addons/html_editor_playground/static/src/playground.js

```diff
@@ -150,7 +150,7 @@ export class Playground extends Component {
     }
 
     get classList() {
-        return this.config.pluginSet === "extras" ? ["odoo-editor-qweb"] : [];
+        return [];
```

Comment:
It will broke some style no ?

Reply:
I've added the class directly in the qweb pluggins, to not be reliant on the plugins set name. It's more robust IMO